lib/entities/workflow.ts

```diff
@@ -61,6 +61,11 @@ export type WorkflowQueryOptions = Omit<PaginationQueryOptions, 'order'> & {
   /** Find workflows containing the specified, comma-separated entities. Requires `sys.entity.sys.linkType` */
   'sys.entity.sys.id[in]'?: string
   'sys.workflowDefinition.sys.id'?: string
+  /**
+   * Order releases by
+   * @default -sys.createdAt
+   * */
+  order?: string
```

Comment:
let's add the enum here with all available options

Reply:
For public-facing APIs, where the value can be a string value that can be passed through from other places, I suggest using a string union over an enum. 

This suggestion might be due to my lack of TS knowledge though 😁 


My understanding is, that whenever you have a property that can be dynamically set, you don't want to enforce an enum transformation.

Reply:
so you mean that I should have something like: 
`type OrderQueryParam = 'sys.updatedAt' | '-sys.updatedAt' | 'sys.createdAt' | '-sys.createdAt'
`order: OrderQueryParam` 
instead of an `enum`?

Reply:
Yes!